hosts.txt

```diff
@@ -155,6 +155,7 @@
 0.0.0.0 flophous.cf
 0.0.0.0 pr0gram.org
 0.0.0.0 authedmine.eu
+0.0.0.0 www.monero-miner.com
```

Comment:
better to have this without www

Reply:
`monero-miner.com` redirect me to `www.monero-miner.com`